src/wrappers/themis/wasm/wasmthemis.mk

```diff
@@ -39,9 +39,7 @@ $(BIN_PATH)/libthemis.js: LDFLAGS += -s LINKABLE=1
 # does not use are missing the from the binary. Suppress the warnings.
 $(BIN_PATH)/libthemis.js: LDFLAGS += -s ERROR_ON_UNDEFINED_SYMBOLS=0
 $(BIN_PATH)/libthemis.js: LDFLAGS += --pre-js $(WASM_PRE_JS)
-
 $(BIN_PATH)/libthemis.js: CMD = $(CC) -o $@ $(filter %.o %a, $^) -O3 $(LDFLAGS)
-
 $(BIN_PATH)/libthemis.js: $(THEMIS_STATIC) $(WASM_RUNTIME) $(WASM_PRE_JS)
```

Comment:
empty lines?

Reply:
It was empty lines.

Reply:
what is the point of this change? why it happened?

<img width="787" alt="Screenshot 2022-08-22 at 20 29 38" src="https://user-images.githubusercontent.com/2877920/185982998-204bafcd-4076-4c96-8ea5-eb4bc80ad03d.png">

Reply:
Cosmetic. I prefer to see one logical thing without new lines.

Reply:
removing empty lines is not a valuable change, it should not be included in this PR